samples/v2/backups-copy.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// sample-metadata:
+//  title: Copies a source backup
+//  usage: node spannerCopyBackup <INSTANCE_ID> <COPY_BACKUP_ID> <SOURCE_BACKUP_ID> <PROJECT_ID>
+
+'use strict';
+
+function main(
+  instanceId = 'my-instance',
+  backupId = 'my-backup',
+  sourceBackupPath = 'projects/my-project-id/instances/my-source-instance/backups/my-source-backup',
+  projectId = 'my-project-id'
+) {
+  // [START spanner_copy_backup]
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const instanceId = 'my-instance';
+  // const backupId = 'my-backup',
+  // const sourceBackupPath = 'projects/my-project-id/instances/my-source-instance/backups/my-source-backup',
+  // const projectId = 'my-project-id';
+
+  // Imports the Google Cloud Spanner client library
+  const {Spanner} = require('@google-cloud/spanner');
+  const {PreciseDate} = require('@google-cloud/precise-date');
+
+  // Creates a client
+  const spanner = new Spanner({
+    projectId: projectId,
+  });
+
```

Comment:
Same add the comment here and everywhere else

Reply:
Done